Разметка главной страницы

Kon3a opened the pull request:
HTML разметка главной страницы проекта Sedona

---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1872463/tasks/3)

:boom: https://htmlacademy-htmlcss.github.io/1872463-sedona-33/2/

Kon3a committed:
Размечает главную страницу

HTML разметка главной страницы проекта Sedona

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+    <header class="page-header">
+      <nav class="navigation">
+         <!-- Логотип Begin -->
+        <a class="logo-link">
```
Логотип не входит в навигацию

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+        </ul>
+        <!-- Основная навигация End -->
+        
+        <ul class="navigation-list navigation-user">
```
В списках должна быть однородная информация, согласно спецификации. Здесь у тебя ссылки  и кнопки - они не однородны. Лучше сделай без списка.

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+            </a>
+          </li>
+          <li class="navigation-item">
+            <button class="button button-navigation">Хочу сюда!</button>
```
Это не кнопка а ссылка. В случае если скрипт не будет работать, тогда по ссылке можно будет перейти на страницу дублер и выполнить тот функционал, который должен выполняться на странице при рабочем скрипте(вызов попапа)

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+        <!-- Welcome End -->
+        <section class="tagline">
+          <h2 class="tagline-title title-h2">
+            Седона — небольшой городок в Аризоне,<br>заслуживающий большего!
```
тут br не нужен
перенос сделаешь при стилизации

Kon3a replied:
Соглашусь с тобой. Но пока не понимаю как это делается стилизацией. Возможно где то что то упустил. Подскажи.

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+        <h1 class="visually-hidden">Город Седона</h1>
+        <!-- Welcome Begin -->
+        <section class="welcome">
+          <h2 class="visually-hidden">Добро пожаловать в Седону</h2>
```
что мешает этот заголовок сделать заголовком первого уровня?

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+        </section>
+        <!-- Преимущества Begin -->
+        <section class="advantages">
+          <ol class="advantages-list">
```
не критично, но тут и обычный список подойде

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+              </p>
+              <img class="advantages-img adv-img-town" src="images/photo-1.jpg" alt="Улица города Седона" width="800" height="384"/>
+              <!-- Дополнительные преимущества/опции Begin -->
+              <ul class="advantages-additional-list">
```
вложенный список - избыточен. просто разбей контент на несколько блоков(по линиям)

Kon3a replied:
Вот здесь я не понимаю что надо сделать. 
Разбить контент по линиям это значит сделать четыре блока дивами?

GirdaViktor commented on index.html:
```diff
@@ -1,12 +1,226 @@
+          <p class="hotel-search-description">
+            Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне
+          </p>
+          <button class="button button-hotel-search">
```
это ссылка

Kon3a committed:
Переносит Логотип

Убрал Логотип из Навигации. Перенёс его в Header

Kon3a committed:
Меняет кнопку на ссылку

Кнопку "Поиск гостиницы в Седоне" заменил на ссылку

Kon3a committed:
Заменяет Список на Дивы

В Пользовательской навигации заменил Список на Дивы.

Kon3a committed:
Заменяет кнопку на ссылку

В Пользовательской навигации заменил кнопку "Хочу сюда на ссылку"

Kon3a committed:
Меняет заголовок H1 и убирает лишние BR

Убрал заголовок первого уровня <h1 class="visually-hidden">Город Седона</h1>,  сделал заголовком первого уровня  "Добро пожаловать в Седону"

Kon3a committed:
Меняет тип списка

Изменил тип списка Преимуществ (advantages-list) с нумерованного на обычный.

Kon3a committed:
Изменяет ссылку

Ссылке "Хочу сюда!" добавил href

Kon3a committed:
Изменяет секцию Преимущества

Создал заголовок для Секции Преимущества (скрытый)

Часть списков убрал.

Сделал преимуществам находящимся на одной линии DIV-обертку.

Пока оставил картинки в <img>, но в дальнейшем при стилизации сделаю их бэкраундом в первом и втором преимуществе,
а у доп.преимуществ Жилье-Еда-Сувениры псевдоэлементами.

Преимущества 3-4-5 пока в виде спискаю. До стилизации. А там посмотрю как удобнее и лучше.

Kon3a committed:
Мелкие правки

Убрал два лишних файла
